examples/computer_vision/susan.cpp

```diff
@@ -0,0 +1,89 @@
+/*******************************************************
+ * Copyright (c) 2015, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#include <cstdio>
+#include <arrayfire.h>
+#include <cstdlib>
+
+using namespace af;
+
+static void susan_demo(bool console)
+{
+    // Load image
+    array img_color;
+    if (console)
+        img_color = loadImage(ASSETS_DIR "/examples/images/square.png", true);
+    else
+        img_color = loadImage(ASSETS_DIR "/examples/images/man.jpg", true);
+    // Convert the image from RGB to gray-scale
+    array img = colorSpace(img_color, AF_GRAY, AF_RGB);
+    // For visualization in ArrayFire, color images must be in the [0.0f-1.0f] interval
+    img_color /= 255.f;
+
+    features feat = susan(img, 3, 32.0f, 10, 0.05f, 3);
+
+    if (feat.getNumFeatures() > 0) {
+        printf("Found features, proceeding ahead\n");
```

Comment:
@pentschev :smile: sure, i will remove it. I just found it rather too silent when it doesn't say anything when no corners are found.